src/test/Makefile

```diff
@@ -238,7 +239,8 @@ telemetry_crsf_msp_unittest_SRC := \
 		$(USER_DIR)/fc/runtime_config.c
 
 telemetry_crsf_msp_unittest_DEFINES := \
-		USE_MSP_OVER_TELEMETRY
+		USE_MSP_OVER_TELEMETRY \
+		NVIC_PriorityGroup_2=0x500
```

Comment:
It may be better to stub this in relevant .c / .h file, something like:
```

#if defined(UNIT_TEST) && !defined(NVIC_PriorityGroup_2)
# define NVIC_PriorityGroup_2 0x500
#endif

Reply:
That's how I had it before and I changed it figuring it would be less impactful in this way :)  I will revert it to how it was before.